From b75d5481a67ef689492e11ce6c0160aeb418c549 Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Mon, 4 Jun 2018 04:21:50 +0200
Subject: [PATCH 04/14] Utils: Always show SIM Settings menu

* This can now write phone numbers to SIM cards, so allow
  single SIM devices to use this menu as well

Change-Id: Icfaf7066b01d49debb204e990d286a1f5aa16d3d
---
 src/com/android/settings/Utils.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/settings/Utils.java b/src/com/android/settings/Utils.java
index 417ac0faeb..aea794cfbb 100644
--- a/src/com/android/settings/Utils.java
+++ b/src/com/android/settings/Utils.java
@@ -792,7 +792,7 @@ public final class Utils extends com.android.settingslib.Utils {
         final TelephonyManager tm =
                 (TelephonyManager) context.getSystemService(Context.TELEPHONY_SERVICE);
 
-        return tm.getSimCount() > 1;
+        return tm.getSimCount() >= 1;
     }
 
     /**
-- 
2.17.1

